### PR TITLE
Add ce_install command to reformat libraries.yaml

### PR DESCRIPTION
**Pull request by partouf**
This PR is so that we can reformat `libraries.yaml` and easily automate adding new libraries to it.

* Adds `underscore_name` by default to everything, the yaml save option mangles the original contents of `underscore_name`
* Adds a `reformat` command to load `libraries.yaml` and save it in the natural order the python dict uses

### CODE REVIEW

File: bin/lib/library_yaml.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+from lib.config_safe_loader import ConfigSafeLoader
+
+class LibraryYaml:
+    def __init__(self, yaml_dir):
+        self.yaml_dir = yaml_dir
+        self.yaml_path = Path(os.path.join(self.yaml_dir, 'libraries.yaml'))
+        self.Load()
+
+    def Load(self):
```
Comment (mattgodbolt) [inline bin/lib/library_yaml.py:13]: can we use python-style naming convention please (horrid `snake_case` but at least consistent with all the rest of the code! thanks!)

File: bin/lib/ce_install.py [hunk 2]
```diff
@@ -281,6 +282,10 @@ def main():
         if num_failed:
             sys.exit(1)
         sys.exit(0)
+    elif args.command == 'reformat':
```
Comment (mattgodbolt) [inline bin/lib/ce_install.py:285]: We could consider more general formatting: `pre-commit` (which we already use) can (re-)format yaml and enforce style, though this opens the door to our own custom stuff. `pre-commit` means it'll automatically happen, so - we could mix and match if we thought it worthwhile.

File: bin/lib/ce_install.py [hunk 3]
```diff
@@ -100,7 +101,7 @@ def main():
                         help='installables must pass any filter (default "False")')
 
     parser.add_argument('command',
-                        choices=['list', 'install', 'check_installed', 'verify', 'amazoncheck', 'build', 'squash', 'squashcheck'],
+                        choices=['list', 'install', 'check_installed', 'verify', 'amazoncheck', 'build', 'squash', 'squashcheck', 'reformat'],
```
Comment (mattgodbolt) [inline bin/lib/ce_install.py:104]: (aside: I really need to move this over to using `click` :))

**Review by mattgodbolt — APPROVED** [review]
LGTM; happy as is but we have other options too :)

File: bin/lib/library_yaml.py (hunk 1 above)
Comment (partouf) [reply bin/lib/library_yaml.py:13]: oops, yes

**Pull request merged**